web-marketplace/src/components/organisms/MediaSection.tsx

```diff
@@ -34,6 +37,7 @@ const MediaSection: React.FC<React.PropsWithChildren<MediaSectionProps>> = ({
         items={
           items?.map((item, i) => (
             <ArticleCard
+              draftText={_(msg`Draft`)}
```

Comment:
Should this use `DRAFT_TEXT` from the constants file?

Reply:
Same.